vshard/CMakeLists.txt

```diff
@@ -8,4 +8,5 @@ add_subdirectory(router)
 # Install module
 install(FILES cfg.lua error.lua consts.lua hash.lua init.lua replicaset.lua
         util.lua rlist.lua heap.lua registry.lua version.lua service_info.lua
+        vshard-version.lua
```

Comment:
Verify that deb and rpm doesn't need any additional updates.

You can use packpack for that one.
```
git clone git@github.com:tarantool/queue.git
OS=debian DIST=bullseye packpack/packpack
OS=fedora DIST=35 packpack/packpack
```